GoMeet/src/main/java/com/google/sps/servlets/UpdateLocationServlet.java

```diff
@@ -23,25 +22,22 @@
 /** Handles fetching and saving location data. */
 @WebServlet("/update-location-data")
 public class UpdateLocationServlet extends HttpServlet {
+
+  private Dao<Location> locationDao = new LocationDao();
+
   /** Accepts a POST request with information about the location entity to add a vote to. */
   @Override
-  public void doPost(HttpServletRequest request, HttpServletResponse response) {
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String keyString = request.getParameter("key");
     try {
-        updateLocation(keyString);
+      locationDao.update(keyString);
```

Comment:
Would be good to send something in the response to verify that the request was successful

Reply:
Yep, sounds like a good idea.